js/src/mpl.js

```diff
@@ -160,7 +157,7 @@ mpl.figure.prototype._init_canvas = function() {
 
     var pass_mouse_events = true;
 
-    canvas_div.resizable({
+    /*canvas_div.resizable({
```

Comment:
Btw, would it make sense to have a phosphor counterpart to `jquery.resizable` with a similar UI (handle in the corner) but also phosphor resize events?